src/test/java/bio/terra/service/job/JobServiceTest.java

```diff
@@ -37,10 +36,11 @@
   private static final Logger logger = LoggerFactory.getLogger(JobServiceTest.class);
 
   private AuthenticatedUserRequest testUser =
-      new AuthenticatedUserRequest()
-          .subjectId("StairwayUnit")
-          .email("stairway@unit.com")
-          .token(Optional.empty());
+      AuthenticatedUserRequest.builder()
+          .setSubjectId("StairwayUnit")
+          .setEmail("stairway@unit.com")
+          .setToken("")
```

Comment:
Just for consistency with the other tests?
```suggestion
          .setToken("token")
```